cloudslang-cli/src/main/java/io/cloudslang/lang/cli/services/SyncTriggerEventListener.java

```diff
@@ -126,6 +123,14 @@ else if (data.containsKey(LanguageEventData.OUTPUTS) && data.containsKey(Languag
                 flowFinished.set(true);
                 printFinishEvent(data);
                 break;
+            case ScoreLangConstants.MAVEN_DEPENDENCY_BUILD:
+                consolePrinter.printWithColor(Ansi.Color.CYAN,
```

Comment:
seems you can extract a two line method for the two newly added cases since the processing is identical